sdk/servicebus/service-bus/samples/typescript/src/browseMessages.ts

```diff
@@ -29,13 +29,13 @@ export async function main() {
   const sbClient = new ServiceBusClient(connectionString);
 
   // If receiving from a subscription you can use the getReceiver(topic, subscription) overload
-  // In this case since we're using `diagnostics.peek()` that any lock mode will work.
-  // `diagnostics.peek()` does not lock messages in either lock mode.
+  // Since browsing messages doesnt take a lock on the message, the receive mode passed to getReceiver
+  // is irrelvant to this sample code
```

Comment:
```suggestion
  // Since browsing messages doesn't take a lock on the message, the receive mode passed to getReceiver
  // is irrelevant to this sample code.
```